CHANGES.rst

```diff
@@ -142,6 +142,7 @@ Improvements
 ^^^^^^^^^^^^
 
 - Add placeholders for accompanying persons to the badge/ticket designer (:pr:`6033`)
+- Add possibility for plugins to add extra fields in a room booking form. Make interceptable hooks after room booking is created and updated (:pr:`6126`)
```

Comment:
This is a bit too long - consider breaking it down:

```suggestion
- Allow plugins to add extra fields to the room booking form (:pr:`6126`)
- Add interceptable hooks after a room booking is created/updated (:pr:`6126`)
```

Reply:
Updated in https://github.com/indico/indico/pull/6126/commits/07b7a5b4c8cfa60b78a1729483d63164aae66801, thanks!